docs/admin-manual/setup/install.md

```diff
@@ -19,19 +19,15 @@ If you are coming from **Airtime**, please follow the [Airtime migration guide](
 You can install LibreTime using the one of the following methods:
 
 - [:rocket: Using the installer](#using-the-installer)
-- :construction: Using Ansible
+- [:construction: Using docker-compose](#using-docker-compose)
```

Comment:
Maybe :rocket:`? My thinking is because the instructions are complete and there, even though they may not be stable

Reply:
Done